### DISCUSSION TITLE
I have two issues while using it and couldn't find the relevant settings. Can anyone help

### DISCUSSION THREAD

**Discussion by zouyq** [hluk/CopyQ#2935] [General]
I enabled word wrapping, but the wrapped content only shows the first line, with the rest being omitted. I want to display more of the wrapped content, including the second and third lines, in the list. How can I achieve this?

In the list of copied content, the font seems to be slightly too high. Which configurations should I modify? I've tried adjusting item_css, but I haven't been able to get it right.
![image](https://github.com/user-attachments/assets/64af01f9-deb4-4582-b696-75bb08061bb3)

**Comment by hluk** [comment]
> I enabled word wrapping, but the wrapped content only shows the first line, with the rest being omitted. I want to display more of the wrapped content, including the second and third lines, in the list. How can I achieve this?

Disable "Show simple items" in History configuration tab (I assume that's the problem).

![image](https://github.com/user-attachments/assets/c7166abf-cfe9-4d5c-990a-5d3df6709054)

> In the list of copied content, the font seems to be slightly too high. Which configurations should I modify? I've tried adjusting item_css, but I haven't been able to get it right.

Looks like that may not be possible to customize using CSS. :thinking:
